File: client/web/sudoo/lib/domain/model/auth/account.dart
import 'package:sudoo/domain/model/auth/provider.dart';

class Account {
  final String? userId;
  final String emailOrPhoneNumber;
  final String password;
  final Provider? provider;
  final bool? isValidate;
  final DateTime? createdAt;

  Account({
    this.userId,
    required this.emailOrPhoneNumber,
    required this.password,
    this.provider,
    this.isValidate,
    this.createdAt,
  });
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/domain/entity/auth/Account.kt
package com.sudo248.sudoo.domain.entity.auth

import java.time.LocalDateTime


/**
 * **Created by**
 *
 * @author *Sudo248*
 * @since 09:02 - 04/03/2023
 */
data class Account(
    val userId: String? = null,
    val phoneNumber: String,
    val password: String,
    val provider: Provider? = null,
    val isValidated: Boolean? = null,
    val createdAt: LocalDateTime? = null,
)
